README.md

```diff
@@ -4002,6 +4002,25 @@ _You can enable the following settings in Xcode by running [this script](resourc
   ```
 
   </details>
+  
+* <a id='remove-empty-extensions'></a>(<a href='#remove-empty-extensions'>link</a>) **Remove empty, non-conforming, extensions.** Does not affect empty extensions that are appended by a macro. [![SwiftFormat: emptyExtension](https://img.shields.io/badge/SwiftFormat-emptyExtension-008489.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#emptyExtension)
```

Comment:
@calda 

Re: https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#emptyExtension

Should I like to the `develop` branch since this is not on `master` yet?

Reply:
So far we always just point to the `master` branch -- this link will be correct once the rule is officially released. Although, maybe we should just change all of our URLs to point to the `develop` branch. Either works for me!